libraries/app/include/graphene/app/full_account.hpp

```diff
@@ -40,8 +40,10 @@ namespace graphene { namespace app {
       bool settle_orders = false;
       bool proposals = false;
       bool assets = false;
-      bool withdraws = false;
-      bool htlcs = false;
+      bool withdraws_from = false;
+      bool withdraws_authorized = false;
```

Comment:
Perhaps `withdraws_to` for consistency with htlc?

Reply:
changed at https://github.com/bitshares/bitshares-core/pull/1749/commits/65d518021bd98250dcb928e29bae3b412072ed57